lib/src/view/design_side_helix.dart

```diff
@@ -62,7 +62,6 @@ class DesignSideHelixComponent extends UiComponent2<DesignSideHelixProps> with P
     // of idx, which is useful for making figures in the documentation showing how the grids work
 //    bool SHOW_HELIX_COORDINATES_INSTEAD_OF_IDX = true;
     bool SHOW_HELIX_COORDINATES_INSTEAD_OF_IDX = false;
-    bool SHOW__INDEX_AND_COORDINATES = props.show_grid_coordinates;
     var temp_text_for_help_doc_figure_making;
```

Comment:
It's fine to use this variable, since it already is set based on the same logic I used for making the figure. But change its name to something more appropriate to its current function. I'd just suggest just `grid_position_str` (i.e., the helix's grid position, as a string).

Also I like to put the explicit type declaration when declaring the variable, if it is not assigned right away, i.e., `String grid_position_str;`. That helps to see its type since it has no initial value to help us see it.